### ISSUE TITLE
Travelling Salesman Problem

### ISSUE THREAD

**Issue by vanshikachokhani** [Algo-Phantoms/Algo-Tree#1760]
**Is your feature request related to a problem? Please describe.**
We are given a set of cities and distances between pair of cities. We need to find the shortest path such that every city is visited only once. and returns to the starting point.

**Describe the solution you'd like**
The path with minimum cost will be found using backtracking approach.

**Do you want to work on it**
I am Gssoc participant and will the solution in C++.

**Comment by OjusWiZard** [comment]
Duplicate of #370
